src/genie/libs/parser/viptela/tests/ShowCloudexpressApplications/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,26 @@
+expected_output = {
+    "index": {
+        0: {
+            "application": "salesforce",
+            "exit_type": "gateway",
+            "gw_sys_ip": "172.16.255.14",
+            "interface": "-",
+            "latency": "103",
```

Comment:
`latency`, `loass` and `vpn` could be integer instead of string

Reply:
Thanks Takashi, I've adjusted based on your feedback. I wasn't sure that these outputs didn't parse a hyphen '-' for a no value rather than a 0. I've made the relevant changes to the schema and the dictionary.

Thanks - Simon